proto/shentu/shield/v1alpha1/genesis.proto

```diff
@@ -23,9 +22,9 @@ message GenesisState {
     string total_collateral = 6 [ (gogoproto.moretags) = "yaml:\"total_collateral\"", (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int", (gogoproto.nullable) = false ];
     string total_withdrawing = 7 [ (gogoproto.moretags) = "yaml:\"total_withdrawing\"", (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int", (gogoproto.nullable) = false ];
     string total_shield = 8 [ (gogoproto.moretags) = "yaml:\"total_shield\"", (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int", (gogoproto.nullable) = false ];
-    string total_claimed = 9 [ (gogoproto.moretags) = "yaml:\"total_claimed\"", (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int", (gogoproto.nullable) = false ];
-    MixedDecCoins service_fees = 10 [ (gogoproto.moretags) = "yaml:\"service_fees\"", (gogoproto.nullable) = false ];    
-    MixedDecCoins remaining_service_fees = 11 [ (gogoproto.moretags) = "yaml:\"remaining_service_fees\"", (gogoproto.nullable) = false ];
+    string total_claimed = 9 [ (gogoproto.moretags) = "yaml:\"total_claimed\"", (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int", (gogoproto.nullable) = false ];   
+    repeated cosmos.base.v1beta1.DecCoin service_fees = 10 [ (gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.DecCoins" ];
+    repeated cosmos.base.v1beta1.DecCoin remaining_service_fees = 11 [ (gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.DecCoins" ];
```

Comment:
I wonder if it's safe to directly change types in the proto file.
A safe way is to leave `MixedDecCoins` there, add  `Fees` instead.

Reply:
Since this is just genesis it should be totally fine